torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -49,8 +70,12 @@ def get_num_channels(image: Union[features.ImageTypeJIT, features.VideoTypeJIT])
 
 
 def get_spatial_size_image_tensor(image: torch.Tensor) -> List[int]:
-    width, height = _FT.get_image_size(image)
-    return [height, width]
+    hw = list(image.shape[-2:])
+    ndims = len(hw)
+    if ndims == 2:
```

Comment:
> Could we maybe define a constant instead ? Also should this be ndims >= 2 ? Just judging from the error message below

Since we do `shape[-2:]` there is no change for this being longer than two elements.

Reply:
Thanks @atalman. Because we explicitly ask for the last 2 dims via `image.shape[-2:]`, it can be only up to `2`. It's possible to define a local constant (within the method) but not a global one because of JIT limitations:
```
RuntimeError: 
python value of type 'int' cannot be used as a value. Perhaps it is a closed over global variable? If so, please consider passing it in as an argument or use a local varible instead.:
  File "./torchvision/prototype/transforms/functional/_meta.py", line 74
def get_spatial_size_image_tensor(image: torch.Tensor) -> List[int]:
    hw = list(image.shape[-TWO:])
                           ~~~ <--- HERE
    ndims = len(hw)
    if ndims == TWO:
'get_spatial_size_image_tensor' is being compiled since it was called from 'get_spatial_size'
  File "./torchvision/prototype/transforms/functional/_meta.py", line 103
def get_spatial_size(inpt: features.InputTypeJIT) -> List[int]:
    if isinstance(inpt, torch.Tensor) and (torch.jit.is_scripting() or not isinstance(inpt, features._Feature)):
        return get_spatial_size_image_tensor(inpt)
               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
    elif isinstance(inpt, (features.Image, features.Video, features.BoundingBox, features.Mask)):
        return list(inpt.spatial_size)
```

Most of our code-base doesn't use constants for shape manipulations. Shall we leave it as-is?